Duplicati/Library/Main/Operation/BackupHandler.cs

```diff
@@ -1183,8 +1183,6 @@ private bool AddBlockToOutput(BackendManager backend, string key, byte[] data, i
                         
                         m_blockvolume.Dispose();
                         m_blockvolume = null;
-                        m_indexvolume.Dispose();

```

Comment:
Calling Dispose() at that point is always an error because the if block above guarantees that m_indexvolume is null at that point.